samples/bluetooth/hci_vs_scan_req/src/main.c

```diff
@@ -0,0 +1,312 @@
+/* main.c - Application main entry point */
```

Comment:
Can you add a babblesim test for this sample?
@cvinayak shows how to do that in the [adv/chain test](https://github.com/zephyrproject-rtos/zephyr/blob/master/tests/bsim/bluetooth/host/adv/chain/CMakeLists.txt#L8)

Reply:
I have no experience with babblesim, I tried adding the `bsim.sh` and the change in `observer.c` like in your commit, then I got an error like this:
`* scan/samples/bluetooth/hci_vs_scan_req/bsim.sh missing copyright`
As I couldn't find another case using `bsim.sh` in the tree, I gave up.